lib/farscape/representor.rb

```diff
@@ -94,7 +94,7 @@ def get_transition(meth, request_params = {}, &block)
     end
 
     def method_transitions
-      transitions.map { |k,v| @agent.client.safe_method?( v.interface_method ) ? {k => v}  : {k+'!' => v} }.reduce(:merge)
+      transitions.map { |k,v| @agent.client.safe_method?( v.interface_method ) ? {k => v}  : {k+'!' => v} }.reduce(:merge) || {}
```

Comment:
The idiom for this is `.reduce( {} , :merge)`, which sets an empty hash as the starting value for the memo.